src/main/java/org/alfresco/utility/data/DataContent.java

```diff
@@ -149,6 +216,53 @@ public FolderModel createFolder()
      * </code>
      */
     public void deleteContent()
+    {
+        AlfrescoHttpClient client = alfrescoHttpClientFactory.getObject();
+
+        if(client.getAlfVersion() >= 5.2)
+        {
+            deleteContentV1RestApi(client);
+        }
+        else
+        {
+            deleteContentCmisApi();
+        }
+    }
+
+    /**
+     * Use this to delete the last resource, either file or folder using V1 RestApi
+     * <code>
+     *  dataContent.usingUser(adminUser).usingSite(siteModel).usingResource(repoFile).deleteContent();
+     * </code>
+     */
+    public void deleteContentV1RestApi(AlfrescoHttpClient client)
+    {
+        // Build request

```

Comment:
again, nice to have the STEP function
"STEP(String.format("DATAPREP: Deleting '%..."

this comment is applicable for all/any other such case

Reply:
Added. Thanks